packages/remote-config/ios/RNFBConfig/RNFBConfigModule.m

```diff
@@ -155,7 +160,7 @@ + (BOOL)requiresMainQueueSetup {
   FIRRemoteConfigActivateCompletion completionHandler = ^(NSError *__nullable error) {
     if(error){
       if(error.userInfo && error.userInfo[@"ActivationFailureReason"] != nil && [error.userInfo[@"ActivationFailureReason"] containsString:@"already activated"]){
-          resolve([self resultWithConstants:@([RCTConvert BOOL:@(YES)]) firebaseApp:firebaseApp]);
+          resolve([self resultWithConstants:@([RCTConvert BOOL:@(NO)]) firebaseApp:firebaseApp]);
```

Comment:
before I forget, I'm going to put the ol' `!` on the PR title because changing return value is breaking, in the release summary we should have as complete/concise a description of exactly what is different and exactly how to alter code as possible, goal being zero issues logged after release (and no extra gray hairs for anyone) :-)